src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/mapper/induction/InPrisonInterestsResourceMapperTest.kt

```diff
@@ -94,9 +94,9 @@ class InPrisonInterestsResourceMapperTest {
     given(instantMapper.toOffsetDateTime(any())).willReturn(modifiedDateTime)
     val expectedResponse = aValidPrisonWorkAndEducationResponse(
       id = inPrisonInterests.reference,
-      inPrisonWork = emptySet(),
+      inPrisonWork = null,
       inPrisonWorkOther = null,
-      inPrisonEducation = emptySet(),
+      inPrisonEducation = null,
```

Comment:
This change (and the one in the neighbouring unit tests below) ensure that we get a null `Set` in the response objects, which makes the API consistent with the create/update requests.